data/sql/updates/pending_db_world/rev_1675618471961745900.sql

```diff
@@ -0,0 +1,124 @@
+(17835,0,1,0,0,0,100,2,8000,12000,15000,17000,0,11,30832,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Kidney Shot'),
+(17835,0,2,0,0,0,100,4,8000,12000,15000,17000,0,11,30832,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Kidney Shot'),
+(17835,0,3,0,67,0,100,2,5000,5000,0,0,0,11,7159,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - Behind Target - Cast Backstab'),
+(17835,0,4,0,67,0,100,4,5000,5000,0,0,0,11,15657,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - Behind Target - Cast Backstab'),
+(17835,0,5,0,0,0,100,4,5000,8000,8000,12000,0,11,38520,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Deadly Poison'),
+(17835,0,6,0,25,0,100,0,0,0,0,0,0,11,31326,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Infinite Assassin - On Reset - Cast Corrupt Medivh');
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=21137 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(21137,0,0,0,6,0,50,1,0,0,0,0,0,1,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Infinite Assassin - On Death - Talk'),
+(21137,0,1,0,0,0,100,2,8000,12000,15000,17000,0,11,14874,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Rupture'),
+(21137,0,2,0,0,0,100,4,8000,12000,15000,17000,0,11,14874,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Rupture'),
+(21137,0,3,0,0,0,100,2,5000,8000,8000,12000,0,11,14873,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Spell Sinister Strike'),
+(21137,0,4,0,0,0,100,4,5000,8000,8000,12000,0,11,14873,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Spell Sinister Strike'),
+(21137,0,5,0,0,0,100,4,5000,8000,8000,12000,0,11,30981,0,0,0,0,0,2,0,0,0,0,0,0,0,0,'Infinite Assassin - In Combat - Cast Crippling Poison'),
+(21137,0,6,0,25,0,100,0,0,0,0,0,0,11,31326,0,0,0,0,0,1,0,0,0,0,0,0,0,0,'Infinite Assassin - On Reset - Cast Spell Corrupt Medivh');
+
+DELETE FROM `creature_text` WHERE `CreatureID`=21137;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(21137,0,0,'More will take my place.',12,0,100,0,0,0,15167,0,'Infinite Assassin'),
+(21137,0,1,'Our time has come!',12,0,100,0,0,0,15163,0,'Infinite Assassin'),
+(21137,0,2,'The wizard will fall!',12,0,100,0,0,0,15164,0,'Infinite Assassin');
+
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1`=2 WHERE `creature_id`=21137;
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1`=3 WHERE `creature_id`=22164;
```

Comment:
This one as well

Reply:
They are Assassins as well but like I said before only 17835 rewards rep

Reply:
These Assasins are the rep versions. Not every Assasin gives rep.